distro/webjar/pom.xml

```diff
@@ -32,17 +32,6 @@
     </dependencies>
   </dependencyManagement>
 
-  <!-- add dependency so this is run after the war was build -->
```

Comment:
I think this has to stay. It was a self-reference, but actually I changed it to refer to the webapp (not webjar!).
I was struggling with the build when I also tried to remove this dependency. I fear it has to stay.
You may not have recognized this since you might have already installed libraries in ~/.m2/repository/org/operaton

Reply:
I tried again with a clean build and it succeeded. Strange. Exactly this dependency caused yesterday troubles. Now it worked with a clean build.
Let's try for now. When we see that it gets unstable then we may need to add the dependency again.